saleor/plugins/webhook/tests/test_list_stored_payment_methods_webhook.py

```diff
@@ -85,6 +89,100 @@ def test_list_stored_payment_methods_with_static_payload(
     )
 
 
+@mock.patch("saleor.webhook.transport.synchronous.transport.cache.get")
+@mock.patch("saleor.webhook.transport.synchronous.transport.send_webhook_request_sync")
+def test_list_stored_payment_methods_subscription_issuing_principal(
```

Comment:
Do we need a separate test for that or that is enough to extend a `subscription_query` for other tests within that file and not create a test that will test just a single field wdyt?

Reply:
If you look at [subscription_queries.py](https://github.com/saleor/saleor/pull/16152/files#diff-53f051aa33a53924f0f33a8922f2dbed773347264e94fa66c1cc6bd551af5020), you'll see these are already splitted like that anyway (mostly per particular event / group of functionally related events).